feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts

```diff
@@ -12,11 +12,11 @@ import { Configurator } from '../../core/model/configurator.model';
   providedIn: 'root',
 })
 export class ConfiguratorStorefrontUtilsService {
-  // TODO(#11681): make keyboardFocusService a required dependency and remove deprecated constructor
   constructor(
     protected configuratorGroupsService: ConfiguratorGroupsService,
     @Inject(PLATFORM_ID) protected platformId: any,
-    protected keyboardFocusService?: KeyboardFocusService
+    @Inject(DOCUMENT) protected document: any,
```

Comment:
Why do we add dependency on DOCUMENT here? I cannot see it's used anywhere in this class

Reply:
I could repace both dependencies: 
```
@Inject(PLATFORM_ID) protected platformId: any,
@Inject(DOCUMENT) protected document: any
```
through `windowRef: WindowRef` as we discussed yesterday.
Is it fine for you?

Reply:
As discussed, I replaced both dependencies:
```
@Inject(PLATFORM_ID) protected platformId: any,
@Inject(DOCUMENT) protected document: any
```
through WindowRef wrapper. 
Furthermore, the constructor deprecation for `ConfiguratorStorefrontUtilsService` was updated accordingly.